settings.gradle.kts

```diff
@@ -1,8 +1,8 @@
 pluginManagement {
     repositories {
         maven {
-            name = 'Fabric'
-            url = 'https://maven.fabricmc.net/'
+            name = "Fabric"
+            url = uri("https://maven.fabricmc.net/")
         }
```

Comment:
```suggestion
maven("https://maven.fabricmc.net/")
```

Reply:
Still preferable to explicitly name this maven as `"fabric"`